deluge/commonoptions.py

```diff
@@ -104,37 +105,42 @@ def __init__(self, *args, **kwargs):
                                 help="Set the log level: %s" % ", ".join(deluge.log.levels))
         self.group.add_argument("-q", "--quiet", action="store_true", default=False,
                                 help="Sets the log level to 'none', this is the same as `-L none`")
-        self.group.add_argument("-r", "--rotate-logs", action="store_true", default=False,
-                                help="Rotate logfiles.")
+        self.group.add_argument("--rotate-log", action="store", nargs="?", const="50M",
+                                help="Enable logfile rotation (optional max file size, default: %(const)s)")
```

Comment:
I think mention either here or in manpage about the backupcount being a set value